src/style/reset.css

```diff
@@ -0,0 +1,7 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
```

Comment:
set it for *

Reply:
isn't fixed
move `box-sizing: border-box;`  to `*` selector